src/app/Main.java

```diff
@@ -69,7 +77,14 @@ public static void main(String[] args) {
     WelcomeView welcomeView = new WelcomeView(switchViewController);
     views.add(welcomeView, WelcomeView.viewName);
 
-    ChatView chatView = new ChatView();
+    OkHttpClient client = new OkHttpClient();
+    var chatViewModel = new ChatViewModel(new ArrayList<>());
+    var messageDataAccessObject = new FirebaseMessageDataAccessObject(client);
+
+    ChatPresenter chatPresenter = new ChatPresenter(chatViewModel);
+    ChatInteractor chatInteractor = new ChatInteractor(chatPresenter, messageDataAccessObject);
+    ChatController chatController = new ChatController(chatViewModel, chatInteractor);
+    ChatView chatView = new ChatView(chatController, chatViewModel);
```

Comment:
We need a UseCaseFactory for this code. 
If we create the Presenter, Interactor, etc. in the main.java file for every use case, then our Main file is going to be too large.